mantle/platform/conf/conf.go

```diff
@@ -398,6 +398,49 @@ func MergeAllV34exp(confObjs []*Conf) (*UserData, error) {
 	return userData, nil
 }
 
+// Config is compressed and added to another via data url
+func ReplaceAndMaybeCompress(config string) (string, error) {
+	originalURL := base64.StdEncoding.EncodeToString([]byte(config))
+	originalSize := len([]byte(originalURL))
```

Comment:
Base64 encoding expands the input data by a consistent percentage (modulo the last few bytes), so the base64 encoding in this function is mostly beside the point.  For clarity, let's just compare the sizes of the unencoded configs.